### DISCUSSION TITLE
HD Play service

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#118] [Show and tell]
I notice that some channels are merged with HD Play, a free service for online view (registration required). 

Like Playmax (other service for local channels), in FAQ from [website](https://www.hdplayonline.com/), says:

> ¿Quién es responsable del contenido que se transmite?
> 
> Cada canal que forma parte de la grilla de HD Play es responsable por la programación que transmite, como así también de los derechos de los contenidos.

So, are the streams from this service or from their websites? If the answer is the second sentence, it could be added to the pe.m3u playlist, as local channels are using it as an alternative to the creation of their web pages or mobile apps.

**Comment by BellezaEmporium** [comment]
I've seen that more and more streams are getting out of Opencaster btw, maybe they're all merging into this website. Hoping it's not the same case as them.

1) It's using 2 different addresses, one for registration/login, the main address from the frontend.
